### PR TITLE
Babylon.js compatibily lists

### PR DESCRIPTION
**Pull request by CedricGuillemet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -223,6 +223,32 @@ This will create a symbolic link in your `node_modules` directory to the `@babyl
 
 For iOS the XCode project needs to be generated with `CMake` as described [above](#ios) and added to your `xcworkspace`.
 
+## Supported React-Native
```
Comment (bghgary) [inline README.md:226]:
```suggestion
## Supported Versions
```

File: README.md [hunk 2]
```diff
@@ -223,6 +223,32 @@ This will create a symbolic link in your `node_modules` directory to the `@babyl
...
+
+The React-native 
+In-between versions not in this list (like 0.66, 0.67, ...) use the previous React-Native version (0.65 in that case)
```
Comment (bghgary) [inline README.md:229]:
```suggestion
When using a React Native version in between the versions listed, use the previous React Native version. For example, React Native version 0.66 or 0.67 should use Babylon React Native versions that correspond to 0.65.
```

File: README.md [hunk 3]
```diff
@@ -223,6 +223,32 @@ This will create a symbolic link in your `node_modules` directory to the `@babyl
...
+
+| React-Native version | BabylonReactNative versions |
```
Comment (bghgary) [inline README.md:231]:
```suggestion
| React Native | Babylon React Native |
```

File: README.md [hunk 4]
```diff
@@ -223,6 +223,32 @@ This will create a symbolic link in your `node_modules` directory to the `@babyl
...
+| 0.64 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
+| 0.65 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
+| 0.69 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
+| 0.70 | 1.5.0, 1.5.1 |
+
+## Supported Babylon.js versions
+
+Depending on the BabylonReactNative NPM package version, some Babylon.js NPM dependency version may or may not be compatible. Here is a compatibility list that has been tested:
+
+| BabylonReactNative version | Babylon.js versions |
```
Comment (bghgary) [inline README.md:242]:
```suggestion
| Babylon React Native | Babylon.js |
```

File: README.md [hunk 5]
```diff
@@ -223,6 +223,32 @@ This will create a symbolic link in your `node_modules` directory to the `@babyl
...
+| React-Native version | BabylonReactNative versions |
+|---|---|
+| 0.64 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
+| 0.65 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
+| 0.69 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
+| 0.70 | 1.5.0, 1.5.1 |
+
+## Supported Babylon.js versions
+
+Depending on the BabylonReactNative NPM package version, some Babylon.js NPM dependency version may or may not be compatible. Here is a compatibility list that has been tested:
```
Comment (bghgary) [inline README.md:240]:
```suggestion
Depending on the Babylon React Native NPM package version, some Babylon.js NPM dependency versions may or may not be compatible. Here is a compatibility list that has been tested:
```

File: README.md [hunk 6]
```diff
@@ -223,6 +223,40 @@ This will create a symbolic link in your `node_modules` directory to the `@babyl
...
+3 NPM Packages are available: 1 mandatory for Windows or Android/iOS, 1 for Windows and 1 for Android/iOS.
+
+Babylon React Native NPM Package naming convention is 
+- `@babylonjs/react-native` for the base package
+- `@babylonjs/react-native-windows-0-XX` for Windows
+- `@babylonjs/react-native-iosandroid-0-XX` for iOS/Android
+
+XX corresponds to the React Native version. `@babylonjs/react-native-windows-0-64` corresponds to React Native 0.64.
+
+## Supported Babylon.js versions
```
Comment (bghgary) [inline README.md:246]: nit:
```suggestion
## Supported Babylon.js Versions
```

File: README.md [hunk 7]
```diff
@@ -223,6 +223,40 @@ This will create a symbolic link in your `node_modules` directory to the `@babyl
...
+| 0.65 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
+| 0.69 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
+| 0.70 | 1.5.0, 1.5.1 |
+
+3 NPM Packages are available: 1 mandatory for Windows or Android/iOS, 1 for Windows and 1 for Android/iOS.
+
+Babylon React Native NPM Package naming convention is 
+- `@babylonjs/react-native` for the base package
+- `@babylonjs/react-native-windows-0-XX` for Windows
+- `@babylonjs/react-native-iosandroid-0-XX` for iOS/Android
```
Comment (bghgary) [inline README.md:242]: FYI, we also have some docs in the package readme: https://github.com/BabylonJS/BabylonReactNative/tree/master/Modules/%40babylonjs/react-native#platform-native-packages

File: README.md [hunk 8]
```diff
@@ -223,6 +223,40 @@ This will create a symbolic link in your `node_modules` directory to the `@babyl
...
+When using a React Native version in between the versions listed, use the previous React Native version. For example, React Native version 0.66 or 0.67 should use Babylon React Native versions that correspond to 0.65.
+
+| React Native | Babylon React Native |
+|---|---|
+| 0.64 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
+| 0.65 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
+| 0.69 | 1.4.0, 1.4.1, 1.4.2, 1.4.3, 1.4.4, 1.5.0, 1.5.1 |
+| 0.70 | 1.5.0, 1.5.1 |
+
+3 NPM Packages are available: 1 mandatory for Windows or Android/iOS, 1 for Windows and 1 for Android/iOS.
```
Comment (nima-ap) [inline README.md:237]: Suggestion (clarity):
`1 mandatory for Windows or Android/iOS` -> `1 mandatory for all platforms`

File: README.md [hunk 9]
```diff
@@ -223,6 +223,40 @@ This will create a symbolic link in your `node_modules` directory to the `@babyl
...
+- `@babylonjs/react-native-windows-0-XX` for Windows
+- `@babylonjs/react-native-iosandroid-0-XX` for iOS/Android
+
+XX corresponds to the React Native version. `@babylonjs/react-native-windows-0-64` corresponds to React Native 0.64.
+
+## Supported Babylon.js Versions
+
+Depending on the Babylon React Native NPM package version, some Babylon.js NPM dependency versions may or may not be compatible. Here is a compatibility list that has been tested:
+
+| Babylon React Native | Babylon.js |
```
Comment (nima-ap) [inline README.md:250]: Suggestion (clarity):
`| Babylon.js |` -> ``| Babylon.js `(@babylonjs/core)` |``

File: README.md [hunk 10]
```diff
@@ -223,6 +223,43 @@ This will create a symbolic link in your `node_modules` directory to the `@babyl
 
 For iOS the XCode project needs to be generated with `CMake` as described [above](#ios) and added to your `xcworkspace`.
 
+## Supported Versions
+
+When using a React Native version in between the versions listed, use the previous React Native version. For example, React Native version 0.66 or 0.67 should use Babylon React Native versions that correspond to 0.65.
```
Comment (bghgary) [inline README.md:228]: This statement isn't necessary anymore right?
```suggestion
```

**Pull request merged**